README.md

```diff
@@ -15,8 +15,28 @@ The gulpfile is 80% from the [gulp-webapp](https://github.com/yeoman/generator-g
 
 There are two commands, both run via npm.
 
-- `npm run build` - clean & build everything and put it into dist folder
-- `npm run serve` - serve the pages and utilize live reload on changes to styles, fonts, images, scripts and HTML.
+- `npm run build` or `gulp build` or `gulp` - clean & build everything and put it into dist folder
```

Comment:
Actually I think that the general gulp is just some kind of shortcut to call the local gulp.
If you don't have the local gulp installed it will throw an error.
```
[14:17:55] Local gulp not found in /guts/projects/procurement
[14:17:55] Try running: npm install gulp
```